docs/environment-variables.md

```diff
@@ -34,6 +34,9 @@ those.
   subgraph if the limit is reached, but will simply restart the syncing step,
   so it can be low. This limit guards against scenarios such as requesting a
   block hash that has been reorged. Defaults to 10.
+- `GRAPH_ETHEREUM_MAX_CONCURRENT_JSON_RPC_CALLS`: The maximum number of
+   requests in-flight at once against Ethereum per batching context.
+   Defaults to 1,000.
```

Comment:
This should mention that this specifically applies to the block ingestor, and not to any other json-roc calls. The env var name should also be more specific.